cylc/flow/scheduler.py

```diff
@@ -451,9 +451,9 @@ async def configure(self, params):
 
         self.workflow_db_mgr.on_workflow_start(self.is_restart)
 
+        self.options.utc_mode = get_utc_mode()
         if not self.is_restart:
             # Set workflow params that would otherwise be loaded from database:
-            self.options.utc_mode = get_utc_mode()
```

Comment:
I think this is wrong - it's overriding what was loaded from the database

Reply:
I think you're right.

Reply:
Confirmed, that was 🤕